### PR TITLE
Zs 1612 tx rollup l2 address

### PR DESCRIPTION
**Pull request by zainen**
all tests passing now with addition for tx_rollup_l2_address

not sure if this is all that needs to be done 😮  feels very minimal right now

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: packages/taquito-michel-codec/src/michelson-validator.ts [hunk 1]
```diff
@@ -417,6 +420,12 @@ export function assertMichelsonInstruction(ex: Expr): ex is MichelsonCode {
         }
         break;
 
+        case "MIN_BLOCK_TIME":
```
Comment (roxaneletourneau) [inline packages/taquito-michel-codec/src/michelson-validator.ts:423]: I am not sure MIN_BLOCK_TIME should be added here as it doesn't take an argument.

File: packages/taquito-michel-codec/src/utils.ts [hunk 2]
```diff
@@ -301,6 +302,7 @@ export const tezosPrefix: Record<TezosIDType, TezosIDPrefix> = {
   P256Signature: [64, [54, 240, 44, 52]], // p2sig(98)
   GenericSignature: [64, [4, 130, 43]], // sig(96)
   ChainID: [4, [87, 82, 0]],
+  tx_rollup_l2_address: [20, [1, 128, 120, 31]]
```
Comment (roxaneletourneau) [inline packages/taquito-michel-codec/src/utils.ts:305]: Consider using PascalCase like for the other keys.

I am a bit confused, I am not sure if it is the correct prefix. From tezos code:
```
let rollup_address =
  {
    b58check_prefix = "\001\128\120\031";
    prefix = "txr1";
    hash_size = 20;
    b58check_size = 37;
  }

let l2_address =
  {
    b58check_prefix = "\006\161\166";
    prefix = "tz4";
    hash_size = 20;
    b58check_size = 36;
  }
```
Does tx_rollup_l2_address refer to a tz4 or txr1 address?

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: packages/taquito-michel-codec/src/utils.ts (hunk 2 above)
Comment (zainen) [reply packages/taquito-michel-codec/src/utils.ts:305]: the contracts that i found in regards to this were with the txr1 address so i used that value but am not sure if its the correct one to use or not

Comment (roxaneletourneau) [reply packages/taquito-michel-codec/src/utils.ts:305]: Ok we can validate that easily.
We can deploy the following contract:
```
parameter tx_rollup_l2_address;
storage unit;
code { DROP; UNIT; NIL operation; PAIR; }
```
and try to call the default entrypoint with a tz4 and a txr1 address and see what is accepted.

Do you want to try?

Comment (zainen) [reply packages/taquito-michel-codec/src/utils.ts:305]: response from the oru-tx channel was that the tz4 are for addresses of account 
txr1 are for addresses of the rollup itself 
tru2 is out of date

**Review by roxaneletourneau — APPROVED** [review]

**Review by e-asphyx — APPROVED** [review]

**Pull request merged**